sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/BeamSqlSeekableTable.java

```diff
@@ -29,6 +31,12 @@ public interface BeamSqlSeekableTable extends Serializable {
   /** prepare the instance. */
   default void setUp() {};
 
+  default void startBundle(
+      DoFn<Row, Row>.StartBundleContext context, PipelineOptions pipelineOptions) {};
```

Comment:
minor - unnecessary semicolon

same as existing setUp() and tearDown()

Reply:
remvoed